### PR TITLE
services endpoint added 

### PR DESCRIPTION
**Pull request by berkguzel**
#### Which issue(s) this PR fixes:
Issue #22  .

#### What this PR does / why we need it:
It creates an endpoint to /resources/services.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -94,6 +95,28 @@ func (c ClusterConnection) GetReplicaSets(clusters []string, namespace string) (
...
+
+	servicesClusterMap := make(map[string][]corev1.Service)
+
+	var errors []error 
+
+	for _, cluster := range clusters {
+		services, err := c.connections[cluster].CoreV1().Services("").List(metav1.ListOptions{})
+		if err != nil {
+			errors = append(errors, err)
+			servicesClusterMap[cluster] = nil
```
Comment (mstrYoda) [inline main.go]: Why do we need to assign nil?

File: main.go [hunk 2]
```diff
@@ -328,3 +357,29 @@ func RollbackDeployment(w http.ResponseWriter, r *http.Request, _ httprouter.Par
...
+	}
+
+	responseBytes, errors := json.MarshalIndent(servicesClusterArray, "", " ")
+	if errors != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return 
+	}
+
+	w.Write(responseBytes)
+	w.WriteHeader(http.StatusOK)
```
Comment (mstrYoda) [inline main.go]: WriteHeader should be called before Write call. Also in this case, we do not need to send StatusOK explicitly. Write already adds it.

**Review by mstrYoda — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (berkguzel) [reply main.go]: I thought that returning nil would be better but yes, it looks unnecessary now.

File: main.go (hunk 2 above)
Comment (berkguzel) [reply main.go]: okay, i will fix it.

**Comment by mstrYoda** [comment]
Thanks for your contribution.

**Review by mstrYoda — APPROVED** [review]

**Pull request merged**